backend/.env.example

```
@@ -32,4 +32,13 @@ TURNSTILE_SECRET_KEY=
 # MODEL_TOKEN=hf_xxxxxxxxxxxxxxxxxxxx
 =========
 # ── CORS ──────────────────────────────────────────────────────────────────────
-CORS_ALLOW_ALL=true
\ No newline at end of file
+CORS_ALLOW_ALL=true
+
+# ── LLM Provider (for AI Chat Assistant) ──────────────────────────────────────
+# Options: gemini (default), openai, claude, ollama
+LLM_PROVIDER=gemini
+GEMINI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GEMINI_API_KEY key should go before the LLM_PROVIDER key

(UnorderedKey)